docs/developer-guide/releases.md

```diff
@@ -23,7 +23,7 @@ The secondary goals are:
         -   If a CHANGELOG item affects only one rule, then it should lead with its name.
         -   The order of items is: Removed, Changed, Deprecated, Added, Fixed.
         -   The items within each group are ordered by the widest reaching first to the smallest, and then alphabetically by rule name.
-    2.  Increment the `version` number in the `package.json` by hand, according to whether it's a patch, minor or major release.
+    2.  Run `npm --no-git-tag-version version major|minor|patch` to increment the `version` number in `package.json` and `package-lock.json`, according to whether it's a patch, minor or major release.
```

Comment:
Oooo, not knowing about that argument is why I haven't been using that command, nice!